stock_exchange/apps/stock/models.py

```diff
@@ -1,4 +1,77 @@
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from datetime import date
 
+from stock_exchange.apps.users.models import User
 
-# Add your models here
\ No newline at end of file
+#This tuple is used for choice option for
+#inform users about market operation status.
+MARKET_OPTIONS = {
+    ('B', 'buy'),
+    ('S', 'sell')
+}
+
+class Currency(models.Model):
+    """
+    Storage for information about used
+    currency in stock.
+    """
+    
+    name = models.CharField(
+        _('Nazwa waluty'),
+        max_length = 100,
+    )
+
+    current_value = models.DecimalField(
+        _('Akutalny kurs'),
+        max_digits = 30,
+        decimal_places = 5
+    )
+
+    description = models.TextField(
+        _('Opis'),
+        blank = True,
+        null = True
+    )
+    
+    def __str__(self):
+        return '%s' % (self.name)
+
+
+class Transactions(models.Model):
+    """
+    Storage for information about all 
+    transaction operations.
+    """
+
+    user = models.ForeignKey(
+        User,
+        on_delete = models.CASCADE,
+        default = ""
```

Comment:
Nani?

Reply:
What's the point of making default with = "" when it's for FK to other model. I'm not sure if this even work. And it's require for transaction to have user also